tests/lib/nested.sh

```diff
@@ -64,18 +88,82 @@ create_nested_core_vm(){
     fi
 }
 
-start_nested_vm(){
-    systemd_create_and_start_unit nested-vm "${QEMU} -m 2048 -nographic -net nic,model=virtio -net user,hostfwd=tcp::8022-:22 -drive file=/tmp/work-dir/ubuntu-core.img,if=virtio,cache=none,format=raw -drive file=${PWD}/assertions.disk,if=virtio,cache=none,format=raw"
+start_nested_core_vm(){
+    systemd_create_and_start_unit nested-vm "${QEMU} -m 2048 -nographic -net nic,model=virtio -net user,hostfwd=tcp::$SSH_PORT-:22 -drive file=$WORK_DIR/ubuntu-core.img,if=virtio,cache=none,format=raw -drive file=${PWD}/assertions.disk,if=virtio,cache=none,format=raw -monitor tcp:127.0.0.1:$MON_PORT,server,nowait -usb"
     if ! wait_for_ssh; then
         systemctl restart nested-vm
     fi
 }
 
-destroy_nested_core_vm(){
+create_nested_classic_vm(){
+    mkdir -p "$WORK_DIR"
+
+    # Get the cloud image
+    IMAGE_URL=$(get_image_url_for_nested_vm)
+    wget -P "$WORK_DIR" "$IMAGE_URL"
+    IMAGE=$(ls $WORK_DIR/*.img)
+
+    # Prepare the cloud-init configuration
+    cat <<EOF > "$WORK_DIR/seed"
+#cloud-config
+  ssh_pwauth: True
+  users:
+   - name: user1
+     sudo: ALL=(ALL) NOPASSWD:ALL
+     shell: /bin/bash
+  chpasswd:
+   list: |
+    user1:ubuntu
+   expire: False
+EOF
+    cloud-localds -H "$(hostname)" "$WORK_DIR/seed.img" "$WORK_DIR/seed"
+
+    # Start the vm
+    start_nested_classic_vm "$IMAGE"
+}
+
+start_nested_classic_vm(){
+    IMAGE=$1
+    QEMU=$(get_qemu_for_nested_vm)
+
+    systemd_create_and_start_unit nested-vm "${QEMU} -m 2048 -nographic -net nic,model=virtio -net user,hostfwd=tcp::$SSH_PORT-:22 -hda $IMAGE -hdb $WORK_DIR/seed.img -monitor tcp:127.0.0.1:$MON_PORT,server,nowait -usb"
```

Comment:
This one too:
```
systemd_create_and_start_unit nested-vm "${QEMU} -m 2048 -nographic \
    -net nic,model=virtio -net user,hostfwd=tcp::$SSH_PORT-:22 \
    -hda $IMAGE -hdb $WORK_DIR/seed.img \
    -monitor tcp:127.0.0.1:$MON_PORT,server,nowait -usb"
```
`-hda`, `-hdb` should be relaced with `-drive file=...,if=virtio..`

Reply:
done